server/plugin.go

```diff
@@ -410,6 +410,7 @@ func (p *Plugin) syncUsers() {
 		return
 	}
 
+	p.metricsService.ObserveUpstreamUsersTotal(int64(len(msUsers)))
 	p.API.LogDebug("Count of MS Teams users", "count", len(msUsers))
```

Comment:
Can we drop this log while we're at it, given it's now redundant? (We can also drop the `Count of MM users` below.)